src/io/flutter/utils/AndroidLocationProvider.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+import com.android.tools.idea.gradle.dsl.model.BuildModelContext;
```

Comment:
We may have issues w/ local development, for people developing against something like IntelliJ CE?

Reply:
Yes, Until we have time to devise a proper solution people can replace AndroidLocationProvider with its stub file content.